tests/config_id/host/host.c

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/tests.h>
+#include "../common.h"
+#include "../host/sgx/cpuid.h"
+#include "config_id_u.h"
+
+static bool _is_kss_supported()
+{
+    uint32_t eax, ebx, ecx, edx;
+    eax = ebx = ecx = edx = 0;
+
+    // Obtain feature information using CPUID
+    oe_get_cpuid(0x12, 0x1, &eax, &ebx, &ecx, &edx);
+
+    // Check if KSS (bit 7) is supported by the processor
+    if (!(eax & (1 << 7)))
+        return false;
+    else
+        return true;
+}
+
+int main(int argc, const char* argv[])
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+
+    if (argc < 3)
+    {
+        fprintf(
+            stderr,
+            "Usage: %s ENCLAVE_PATH [no_kss_flags|with_kss_flags] "
+            "[--host-threads n] [--enclave-threads n] "
```

Comment:
are these options relevant?

Reply:
not sure about [--host-threads n] [--enclave-threads n]  ..I continued it from previous code